### PR TITLE
Simple image upload enabled.

### PR DESCRIPTION
**Pull request by Hyper-Node**
# MaRDI Pull Request
(this template is to be filled out by the person who is creating the pull-request)

**Goal**: 
(bigger goal you want to achieve with this PR)
- Enable simple image upload, which seems ok for some images in documentation 
- Works fine locally

**Changes**:
(summary of the basic changes to the source-files within this PR)
- activate setting,  data types for image upload and 

**Instructions for PR review**:
(these are marked by the person creating the PR and should be checked by the reviewer) 
- [x] Conceptual Review (Logic etc...) 
- [x] Code Review (Review your implementation) 
- [ ] Checkout (Test changes locally) 
Requesting some considerations to folder path, this could be a mounted volume to have persistent images. Changing image mount is rather a proposal. 

**Checklist for this PR**: 
- [x] At least one reviewer has been assigned. 
- [x] All related issues with further descriptions are linked.

**Note for linking-issue across repositories**:
If the issue is not in the same Repository as the PR. The issue can be referenced with by 
typing the following here in this markdown: 
user/repository#number
https://github.com/MaRDI4NFDI/portal-compose/issues/32

### CODE REVIEW

**Comment by Hyper-Node** [comment]
had to run $sudo chmod -R 774 locally to make it work.

File: docker-compose.yml
Comment (Hyper-Node) [inline docker-compose.yml:15]: removed it since this is not overwritten by values in env file.

Comment (aot29) [reply docker-compose.yml:15]: The way things are deployed, all defaults should be in docker-compose.yml. So if you remove this, WIKIBASE_HOST will not be set on the production instance.

File: mediawiki/LocalSettings.d/LocalSettings.override.php
Comment (aot29) [inline mediawiki/LocalSettings.d/LocalSettings.override.php:76]: If you add Lockdown but it's not on the server, it will crash the portal

**Comment by aot29** [comment]
I had to run `chown -R www-data /var/www/html/images` inside the wikibase container. Maybe it would be necessary to add an entrypoint to the wikibase container and run something like this.

File: .gitignore
Comment (Hyper-Node) [inline .gitignore:9]: ok, looks reasonable if we have more defaults we might just create images/defaults or something and set this to non-ignore

**Review by aot29 — APPROVED** [review]
OK but as we discussed before don't merge into main just yet

**Comment by Hyper-Node** [comment]
Closing this seet private docuemnation AND image upload for all changes.

**Pull request closed without merging**
